headers/entryHeader.js

```diff
@@ -23,8 +23,6 @@ module.exports = function () {
     switch(process.platform){
         case 'win32':
             _verMade |= 0x0A00;
-        case 'darwin':
-            _verMade |= 0x1300;
```

Comment:
It seems macOS and OSX seems to use linux file permissions instead. So removing OSX case here would be acceptable.